gnomad/utils/file_utils.py

```diff
@@ -54,70 +52,6 @@ def select_primitives_from_ht(ht: hl.Table) -> hl.Table:
                         v.dtype in {hl.tstr, hl.tint32, hl.tfloat32, hl.tint64, hl.tfloat64, hl.tbool}})
 
 
-def ht_to_vcf_mt(
```

Comment:
Yeah I guess it does make since to move this to a `utils.vcf` because it is specific to writing to a vcf and since we are planning to add more to the vcf module at somepoint. Good catch